pkg/apis/resolution/v1beta1/resolution_request_types.go

```diff
@@ -63,7 +63,8 @@ type ResolutionRequestSpec struct {
 	// path to file, the kind of authentication to leverage, etc.
 	// +optional
 	// +listType=atomic
-	Params []pipelinev1.Param `json:"params,omitempty"`
+	Params         []pipelinev1.Param `json:"params,omitempty"`
+	ResolutionName string             `json:"resolutionName,omitempty"`
```

Comment:
Could we name it `url` instead ? at lesat on the the resolution part `ResolutionRequest`, `Resolve` function, … ? (I don't remember entirely where did we land in the TEP)

Reply:
Yes. I think we decided on `name` in the user spec. But internally, we can change it to `url` for the `ResolutionRequest`. It also provides better readability.

Reply:
Fixed!